src/OpenTracing/Noop/NoopTracerFactory.cs

```diff
@@ -0,0 +1,10 @@
+namespace OpenTracing.Noop
+{
+    public static class NoopTracerFactory
+    {
+        public static ITracer Create()
```

Comment:
[nit] May be useful to add comment to the publicly visible method. That also gives us a method to communicate to the developer that this will return a singleton instance.